portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -23,13 +23,15 @@
   private final String message;
   private final long timestamp;
   private final double score;
+  private final boolean usercomment;
 
-  public Comment(long id, String name, String email, String message, long timestamp, double score) {
+  public Comment(long id, String name, String email, String message, long timestamp, double score, boolean usercomment) {
```

Comment:
add javadoc (method level comment in javadoc format, explaining the input parameter), also as the list grows you might want to use Builder pattern,